autopush/router/gcm.py

```diff
@@ -115,19 +115,24 @@ def _route(self, notification, uaid_data):
         try:
             gcm = self.gcm[router_data['creds']['senderID']]
             result = gcm.send(payload)
+        except RouterException:
```

Comment:
these 2 lines are redundant

Reply:
No, they're not. There's a general Exception handler further down that transposes unexpected exceptions into RouterExceptions with logging.